src/redux/features/funder/projects/funderProjectsSelectors.ts

```diff
@@ -0,0 +1,4 @@
+import { RootState } from 'src/redux/rootReducer';
+
+export const selectFunderProjectRequest = (projectId?: number) => (state: RootState) =>
```

Comment:
Not sure if it has any advantage or not, but I think we always make the param here (in the selector) required, and where the selector is called we pass -1 if there is no selection (no projectId selected in this case)